### ISSUE TITLE
Heading implementation

### ISSUE THREAD

**Issue by thibauddavid** [jordond/compass#135]
Hi,

Compass provides access to course, but doesn't allow tracking heading information.
I think having the ability to get it should be in the scope of that library.

Any plan about this ?

**Comment by jordond** [comment]
You can use the `Azimuth`, but I suppose it may not be exactly what you need. Please discuss in #226

**Comment by thibauddavid** [comment]
Nope, azimuth is indeed different. I made my own implementation which better suits my needs.
Thanks for answering though
